addons/test_mail/static/tests/attachment_view.test.js

```diff
@@ -0,0 +1,137 @@
+import { defineTestMailModels } from "@test_mail/../tests/test_mail_test_helpers";
+import { describe, test, expect } from "@odoo/hoot";
+import { queryOne, waitUntil, click as hootClick } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import {
+    click,
+    contains,
+    openFormView,
+    registerArchs,
+    start,
+    startServer,
+    patchUiSize,
+    SIZES,
+} from "@mail/../tests/mail_test_helpers";
+import { browser } from "@web/core/browser/browser";
+import { patchWithCleanup } from "@web/../tests/web_test_helpers";
+
+describe.current.tags("desktop");
+defineTestMailModels();
+
+/*
+ * This test makes sure that the attachment view controls are working in the following cases:
+ * - Before opening the popout window
+ * - Inside the popout window
+ * - After closing the popout window
+ */
+test("Attachment view test", async () => {
+    const popoutIframe = document.createElement("iframe");
+    const popoutWindow = {
+        closed: false,
+        get document() {
+            const doc = popoutIframe.contentDocument;
+            const originalWrite = doc.write;
+            doc.write = (content) => {
+                // This avoids duplicating the test script in the popoutWindow
+                const sanitizedContent = content.replace(
+                    /<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi,
+                    ""
+                );
+                originalWrite.call(doc, sanitizedContent);
+            };
+            return doc;
+        },
+        get body() {
+            return popoutWindow.document.querySelector(".o_popout_wrapper");
+        },
+        close: () => {
+            popoutWindow.closed = true;
+            popoutIframe.remove(popoutWindow.body);
+        },
+        contains: async (selector) => {
+            await animationFrame();
+            await waitUntil(() => popoutWindow.body);
+            const target = popoutWindow.body.querySelector(selector);
+            expect(target).toBeDisplayed();
+            return target;
+        },
+        click: async (selector) => {
+            const target = await popoutWindow.contains(selector);
+            hootClick(target);
+        },
```

Comment:
Why `hootClick` rather than mail `click`?

If no reason, keep using mail `click`, so less important and consistency in whole file.
There's plan to convert mail test utils to web test utils, but due to lack of expressiveness in tests with web utils that are important for mail (e.g. `await contains("input", { value: "" })` that checks input is empty, not that there's a value) that's why we still use mail test helpers for the time being.

Don't mix `web` and `mail` without a genuine motivation, because this makes the test harder to maintain for no good reasons. It's easy to convert all mail utils to web utils when the time comes.